component/main.jsonnet

```
@@ -4,14 +4,31 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 
 local params = inv.parameters.crossplane;
+local on_openshift4 = inv.parameters.facts.distribution;
```

We generally make `on_openshift4` a boolean, so we can later on use it as just `if on_openshift4 then ...`.

```suggestion
local on_openshift4 = inv.parameters.facts.distribution == 'openshift4';
```